lib/change_the_subject.rb

```diff
@@ -1,23 +1,24 @@
 # frozen_string_literal: true
 
 require "yaml"
+require "pry-byebug"
```

Comment:
Doesn't this re-introduce the pry-byebug problem?

Reply:
I use byebug a lot as a tool during test driven development - is there a way to make it consistently available in dev / test without running the risk of accidentally re-introducing it as a production dependency?

Reply:
Good question that's worth looking into. Fixed now.